src/test/java/pages/Users.java

```diff
@@ -0,0 +1,31 @@
+package pages;
+
+public class Users {
+    protected String firstName,
```

Comment:
All variables should be a private and access to variables should achieve using getters and setters